ecl/hql/hqlexpr.hpp

```diff
@@ -726,6 +726,7 @@ enum node_operator : unsigned short {
         no_merge_pending,
         no_httpcall,
         no_getenv,
+        no_getsecretvalue,
```

Comment:
Needs to be added to the end of the enumeration (before no_last_op), or replace one of the no_unusedNNN.